jest.config.js

```diff
@@ -43,6 +43,7 @@ module.exports = {
   modulePathIgnorePatterns: [
     './app/cdap/components/AbstractWidget/SchemaEditor/Context/__tests__/schemas.js',
     './app/cdap/components/AbstractWidget/SchemaEditor/Context/__tests__/data',
+    './selenium/integration/tethering',
```

Comment:
So we're ignoring the tethering tests for now?

Reply:
yes they need their own setup (i.e., a cdf instance + an hdf cluster). They'll probably move to their own repo once the test setup is done by the backend team.